core/src/main/scala/tech/beshu/ror/accesscontrol/logging/ObfuscatedHeaderShowFactory.scala

```diff
@@ -0,0 +1,15 @@
+package tech.beshu.ror.accesscontrol.logging
+
+import cats.Show
+import cats.implicits._
+import tech.beshu.ror.accesscontrol.domain.Header
+import tech.beshu.ror.accesscontrol.show.logs._
+
+object ObfuscatedHeaderShowFactory {
+  def create(obfuscatedHeaders: Set[Header.Name]): Show[Header] = {
```

Comment:
please move it close to other Show typeclass implementations (tech.beshu.ror.accesscontrol.show.logs)